src/App.tsx

```diff
@@ -45,6 +46,10 @@ const App: React.FC = () => {
     }
   }, [account, connect])
 
+  useEffect(() => {
+    document.title = ["PancakeSwap", cakePriceUsd.toFixed(3)].join(' - ')
+  }, [cakePriceUsd])
+
```

Comment:
Perfect. Just 4 more minor changes :)

1. Put the hook into it's own file (in the hooks folder)
2. Please use [toLocaleString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString)
3. Add a `$` sign
4. Use a template literal instead of the join

```
`PancakeSwap - $${cakePriceUsd}`
```

Reply:
Thank you for the feedback, I just committed the changes requested. Please let me know if this is what you had in mind.